src/Pages/Overview/DisplayProfile.js

```diff
@@ -0,0 +1,101 @@
+import React from 'react'
+import './App.css'
+const enums = require('../../Enums.js')
+
+let toggle = false
+let toggleDelete = false
+let userToDelete = null
+let userToEdit = null
+
+// displaying children of member-board
+export function displayBoard (user, index) {
+  return (
+    <tr key={index}>
+      <td>
+        {user.firstName[0].toUpperCase() +
+          user.firstName.slice(1, user.firstName.length) +
+          ' ' +
+          user.lastName[0].toUpperCase() +
+          user.lastName.slice(1, user.lastName.length)}
+        {user.middleInitial.trim() !== '' &&
+          ' ' + user.middleInitial.toUpperCase() + '.'}
+      </td>
+
+      <td>{user.doorCode}</td>
+
+      <td>{user.pagesPrinted}/30</td>
+
+      <td>{this.marks(user.emailVerified)}</td>
+
+      <td>{enums.getKey(enums.membershipStatus, user.accessLevel)}</td>
+
+      <td>
+        <button
+          className='delete'
+          onClick={() => {
+            userToDelete = user
+            togglerDelete()
+          }}
+        >
+          <svg width='24' height='24' viewBox='0 0 24 24'>
+            <path d='M3 6l3 18h12l3-18h-18zm19-4v2h-20v-2h5.711c.9 0 1.631-1.099 1.631-2h5.316c0 .901.73 2 1.631 2h5.711z' />
+          </svg>
+        </button>
+      </td>
+
+      <button
+        className='delete'
+        onClick={() => {
+          userToEdit = user
+          toggler()
+        }}
+      >
+        <svg width='40' height='40' viewBox='0 0 24 24'>
```

Comment:
I have no idea what this SVG is supposed to show when i look at the code. what would be good is to put each SVG in it's own component, for example `<GreenCheck />` is much easier to understand and import when needed